terraform/concourse/rds.tf

```diff
@@ -0,0 +1,47 @@
+provider "aws" {
+  region = "${var.region}"
+}
+
+resource "aws_db_subnet_group" "concourse_rds" {
+    name = "${var.env}-concourse-rds-subnet"
```

Comment:
let's use only 2 spaces for intendation, like everywhere else in tf files, not 4